webapp/pages/group/edit/_id.vue

```diff
@@ -12,7 +12,7 @@
       </ds-flex-item>
     </ds-flex>
     <ds-space centered>
-      <nuxt-link to="/group/my-groups">zurück</nuxt-link>
+      <nuxt-link to="/my-groups">zurück</nuxt-link>
```

Comment:
locale?

Reply:
> locale?

I don't understand …

@ogerly forgot to change this at that time! So I did.